packages/mantine/src/form/use-form.ts

```diff
@@ -0,0 +1,31 @@
+import {useForm as useMantineForm} from '@mantine/form';
+import {_TransformValues, UseFormInput, GetInputProps} from '@mantine/form/lib/types';
+
+export const useForm = <
```

Comment:
Is this task still relevant?
If so, we would also need to override the [FormProvider](https://github.com/mantinedev/mantine/blob/master/src/mantine-form/src/FormProvider/FormProvider.tsx) in this pull request so that it can use the useForm overload.